docs/glossary.md

```diff
@@ -1,11 +1,13 @@
 ## Wikijump Glossary
 
 There are a number of concepts within Wikidot, which can take on a number of different names in casual usage.
-To standardize their usage during development and within the API, this document will describe the standard name, Wikidot's name (if different), and any other aliases.
+To standardize their usage during development and within the API, this document will describe the standard API name, Wikidot's name (if different), and any other aliases.
+
+Please use the Wikijump API term when writing technical documentation, writing code, or utilizing the API. The aim is clarity and disambiguity.
```

Comment:
```suggestion
Please use the Wikijump API term when writing technical documentation, writing code, or utilizing the API. The aim is clarity and avoiding ambiguity.
```

I'm not certain disambiguity is a word: the top result on google is a post from 2008 from a blog of the same name investigating whether it's a word.

Reply:
It's definitely a word, but I'm fine with this suggestion.